taskfiles/k8s.yml

```diff
@@ -242,21 +267,21 @@ tasks:
       - cmd: docker login --username {{.DOCKERHUB_USER}} --password {{.DOCKERHUB_TOKEN}}
         silent: true
       - docker buildx build
-          --platform linux/arm64,linux/amd64
-          --file operator/Dockerfile
-          --target manager
-          --tag docker.io/vectorized/{{.OPERATOR_REPO}}:{{.TAG_NAME}}
-          --tag docker.io/redpandadata/{{.OPERATOR_REPO}}:{{.TAG_NAME}}
-          --push
-          ./dist
+        --platform linux/arm64,linux/amd64
+        --file operator/Dockerfile
+        --target manager
+        --tag docker.io/vectorized/{{.OPERATOR_REPO}}:{{.TAG_NAME}}
+        --tag docker.io/redpandadata/{{.OPERATOR_REPO}}:{{.TAG_NAME}}
+        --push
+        ./dist
       - docker buildx build
-          --platform linux/arm64,linux/amd64
-          --file operator/Dockerfile
-          --target configurator
-          --tag docker.io/vectorized/{{.CONFIGURATOR_REPO}}:{{.TAG_NAME}}
-          --tag docker.io/redpandadata/{{.CONFIGURATOR_REPO}}:{{.TAG_NAME}}
-          --push
-          ./dist
+        --platform linux/arm64,linux/amd64
+        --file operator/Dockerfile
+        --target configurator
+        --tag docker.io/vectorized/{{.CONFIGURATOR_REPO}}:{{.TAG_NAME}}
+        --tag docker.io/redpandadata/{{.CONFIGURATOR_REPO}}:{{.TAG_NAME}}
```

Comment:
do we need to do an equivalent `-nightly` repo for the configurator images?

Reply:
I will work on the changes to operator helm chart, so that operator container image could be used in mode v1 (old operator).